src/DwarfMap.cpp

```diff
@@ -116,20 +113,32 @@ void DwarfMap::retrieve_die_data(Dwarf_Die die) {
     Dwarf_Off  ptr_address = 0;
 
 
-    int got_name = !dwarf_diename(die,&name,&error);
-    int got_line = !dwarf_attr(die, DW_AT_decl_line, &attr, &error) && !dwarf_formudata(attr, &in_line, &error);
-    int got_file = !dwarf_attr(die, DW_AT_decl_file, &attr, &error) && !dwarf_formudata(attr, &in_file, &error);
+    dwarf_diename(die,&name,&error);
+    !dwarf_attr(die, DW_AT_decl_line, &attr, &error) && !dwarf_formudata(attr, &in_line, &error);
+    !dwarf_attr(die, DW_AT_decl_file, &attr, &error) && !dwarf_formudata(attr, &in_file, &error);
     int got_loclist = !dwarf_hasattr(die, DW_AT_location, &bAttr, &error) && !dwarf_attr(die, DW_AT_location,
         &attr, &error)
 	    && !dwarf_loclist(attr, &loc_list, &num_loc, &error);
-    int got_tag_name = !dwarf_tag(die,&tag,&error) && dwarf_get_TAG_name(tag,&tagname);
+    !dwarf_tag(die,&tag,&error) && dwarf_get_TAG_name(tag,&tagname);
 
-    if(got_loclist && loc_list[0].ld_cents == 1 && got_name && got_file && got_line && got_tag_name) {
+    //printf("%d %d %d %d %d\n", got_name, got_line, got_file, got_loclist, loc_list[0].ld_cents);
```

Comment:
Are the added commented out lines here and below useful? If not, they should be removed.

Reply:
Nope! They can be removed!